lib/hooks/useLoadFlag.ts

```diff
@@ -0,0 +1,9 @@
+import useSWR from "swr";
+import axios from "axios";
+
+const fetcher = (url: string) => axios.get(url).then((response) => response.data);
+
+export const useLoadFlag = (key: string): { isLoading: boolean; status: boolean | undefined } => {
```

Comment:
Note we can update useFlag to use this code but wanted to verify the approach first.